caab-service/src/integrationTest/java/uk/gov/laa/ccms/data/api/controller/ApplicationControllerIntegrationLocalTest.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.laa.ccms.data.api.controller;
+
+import static org.springframework.test.context.jdbc.Sql.ExecutionPhase.AFTER_TEST_METHOD;
+import static org.springframework.test.context.jdbc.Sql.ExecutionPhase.BEFORE_TEST_METHOD;
+import static org.springframework.test.context.jdbc.SqlMergeMode.MergeMode.MERGE;
+
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.jdbc.SqlMergeMode;
+import uk.gov.laa.ccms.caab.api.CaabApiApplication;
+
+@SpringBootTest(classes = CaabApiApplication.class)
+@SqlMergeMode(MERGE)
+@ActiveProfiles("local")
+@Sql(executionPhase = AFTER_TEST_METHOD, scripts = "/sql/delete_data.sql")
```

Comment:
Does it need to run before and after?

Reply:
If you have any local data on the db that has the same ids as used in the tests they could fail.